manager-parent/manager-core/src/test/java/com/redhat/service/smartevents/manager/core/dns/openshift/DnsServiceOpenshiftImplTest.java

```diff
@@ -27,6 +29,8 @@
 import static org.mockito.Mockito.when;
 
 @QuarkusTest
+// This annotation also bootstraps Quarkus into using DevServices for ALL @QuarkusTest's
+@QuarkusTestResource(PostgresResource.class)
```

Comment:
Is this on purpose or you were looking for `restrictToAnnotatedClass = true`?

Reply:
Without the annotation Quarkus was complaining about there being an undefined DataSource.

I had moved a class from `manager-core` to `manager-v1` that had the annotation so I assumed its existence had allowed tests in `manager-core` to work previously. Adding the annotation to a(ny) test in `manager-core` reinstated successful execution of the tests.

Reply:
I dug a bit more...

It's needed for our DAO tests and Quartz.

I've moved the annotation from the Dns test to a DAO test where its inclusion makes more sense.